app/clients/hedera/hedera.go

```diff
@@ -1,3 +1,58 @@
 package hedera
 
-// TODO hedera client for interacting with hedera nodes
+import (
+	"encoding/json"
+	"fmt"
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/model/transaction"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type Client struct {
+	mirrorAPIAddress string
+	mirrorClient     *hederasdk.MirrorClient
+	httpClient       *http.Client
+}
+
+func (hedera Client) GetMirror() *hederasdk.MirrorClient {
+	return hedera.mirrorClient
+}
+
+func NewClient(mirrorAPIAddress, mirrorNodeClientAddress string) *Client {
+	mirrorClient, e := hederasdk.NewMirrorClient(mirrorNodeClientAddress)
+	if e != nil {
+		log.Printf("Error: Could not instantiate mirror client - [%s]", e)
+	}
+
+	return &Client{
+		mirrorAPIAddress: mirrorAPIAddress,
+		mirrorClient:     &mirrorClient,
+		httpClient:       &http.Client{},
+	}
+}
+
+func (hedera Client) GetTransactionsByAccountIdAndTimestamp(account hederasdk.AccountID, lastProcessedTimestamp string) (*transaction.Transactions, error) {
```

Comment:
I think the `account` parameter should be named `accountId`.
I think the `lastProcessedTimestamp` parameter is very context-specific name. I would name it `milestoneTimestamp` or something indicating a specific point in time.

Reply:
As timestamps are numbers, I would expect the timestamp parameter to be some kind of `unit`, not `string`. This way we are forcing conversion for the person using the function